Sources/DifferentiableSwiftExamplesDocumentation/DifferentiableSwiftExamples.docc/Setup.md

```diff
@@ -0,0 +1,41 @@
+# Setup
+
+How to setup your environment and get started with differentiable Swift.
+
+## Overview
+
+In stable Swift releases the Differentiable Swift features has been disabled. So in order to use Differentiable Swift you will need to install a development snapshot version of the toolchain. There are multiple way to download, install and manage different versions of toolchains depending on your needs and/or platform. 
+
+After a toolchain has been installed there are a few things to take into consideration in order to successfully run your differentiable Swift code.
+
+### Installing a toolchain
+
+Toolchains can be manually downloaded and installed from [swift.org](https://swift.org/download). If you do so make sure to select a development snapshot and not a stable release since Differentiable Swift is dissabled on the stable branches. 
```

Comment:
All Swift.org toolchains have the necessary `_Differentiation` module, so they can use any of them. The advantage with the nightlies is that they may have more features or better performance.